file/file_prefetch_buffer.cc

```diff
@@ -676,6 +682,9 @@ bool FilePrefetchBuffer::TryReadFromCacheUntracked(
     } else {
       return false;
     }
+  } else if (!for_compaction) {
+    RecordTick(stats_, PREFETCH_HITS);
+    RecordTick(stats_, PREFETCH_BYTES_USEFUL, n);
```

Comment:
Or this could be added at Line 692. That'll include for both cases - 1. data already present in buffer and 2. went for prefetching.